tests/tcp-test.py

```
@@ -7,12 +7,12 @@
 import meshtastic
 
 
-def onConnection(
-    interface, topic=pub.AUTO_TOPIC
+def on_connection(
+    iface, topic=pub.AUTO_TOPIC
```

Ruff (0.15.6)

[warning] Unused function argument: `topic`

(ARG001)